src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
When I tried prettier on Normalizr, this is the one change that annoyed me to no end. I'm sure I could get used to it,  but the latter is the style that we've been using for so long that it's just too ingrained in habit.